test/integration/klusterlet_test.go

```diff
@@ -582,6 +587,39 @@ var _ = ginkgo.Describe("Klusterlet", func() {
 				return nil
 			}, eventuallyTimeout, eventuallyInterval).ShouldNot(gomega.HaveOccurred())
 		})
+
+		ginkgo.It("Deployment should be added hostAliases when add hostAlias into klusterlet", func() {
+			_, err := operatorClient.OperatorV1().Klusterlets().Create(context.Background(), klusterlet, metav1.CreateOptions{})
```

Comment:
nit: shouldn't hostAliase configuration set here? it would not impact other cases

Reply:
I will set hostAlias configuration here  only for this case.